webpack/assets/javascripts/react_app/components/PF4/Bookmarks/BookmarksActions.js

```diff
@@ -0,0 +1,14 @@
+import URI from 'urijs';
+import { get } from '../../../redux/API';
+import { BOOKMARKS } from './BookmarksConstants';
+
+export const getBookmarks = (url, controller) => {
+  const uri = new URI(url);
+  // eslint-disable-next-line camelcase
+  uri.setSearch({ search: `controller=${controller}`, per_page: 100 });
```

Comment:
I wonder what would happen if there would really be 100 bookmarks :thinking:

Reply:
Since this PR was created there was added a `per_page: 'all'` option here: https://github.com/theforeman/foreman/pull/8304
So I'll change to that :)